scripts/common/copy_config_files.sh.in

```diff
@@ -30,15 +30,16 @@ sudo mkdir -p /var/log/stratum/
 
 SOURCE_DIR=${P4CP_INSTALL}/share/stratum/@TARGET_NAME@
 TARGET_DIR=/usr/share/stratum/@TARGET_NAME@
+TLS_CERTS_DIR=/usr/share/stratum
 
 sudo mkdir -p ${TARGET_DIR}
 sudo cp ${SOURCE_DIR}/@TARGET_NAME@_port_config.pb.txt ${TARGET_DIR}/
 sudo cp ${SOURCE_DIR}/@TARGET_NAME@_skip_p4.conf ${TARGET_DIR}/
 
 #... Install files required for TLS certificate generation ...#
-sudo cp ${SOURCE_DIR}/ca.conf ${TARGET_DIR}/
-sudo cp ${SOURCE_DIR}/grpc-client.conf ${TARGET_DIR}/
-sudo cp ${P4CP_INSTALL}/sbin/generate-certs.sh ${TARGET_DIR}/
+sudo cp ${SOURCE_DIR}/ca.conf ${TLS_CERTS_DIR}/
+sudo cp ${SOURCE_DIR}/grpc-client.conf ${TLS_CERTS_DIR}/
+sudo cp ${P4CP_INSTALL}/sbin/generate-certs.sh ${TLS_CERTS_DIR}/
```

Comment:
Why are we copying `generate-certs.sh` to `/usr/share/stratum`?

It's an administrative script. It belongs in the `sbin` directory.

Reply:
Resolved after offline discussion with Derek